database/fake_data.sql

```diff
@@ -45,35 +45,27 @@ VALUES
 
 INSERT INTO image (image_name, image_url, image_date, region_id, constellation_id)
 VALUES
-('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-01', 1, 1),
-('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-02', 2, 2),
-('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-03', 3, 3),
-('ASTRO_moon', 'https://upload.wikimedia.org/wikipedia/commons/thumb/e/e1/FullMoon2010.jpg/1200px-FullMoon2010.jpg', '2024-10-04', 4, 4),
-('ASTRO_moon', 'https://upload.wikimedia.org/wikipedia/commons/thumb/e/e1/FullMoon2010.jpg/1200px-FullMoon2010.jpg', '2024-10-05', 5, 5),
+('NASA_apod', 'https://science.nasa.gov/wp-content/uploads/2023/09/Carina_Nebula-1.jpeg', '2024-10-01', NULL, 1),
```

Comment:
Good to have NULL values as these aren't linked to a region

Reply:
As discussed, NULL values are accepted by the schema and actually necessary so the expected numbers are passed in. Needed addition, LGTM.